From: =?utf-8?q?Jeremy_B=C3=ADcha?= <jeremy.bicha@canonical.com>
Date: Sun, 16 Mar 2025 19:58:25 -0400
Subject: Skip multipage test that fails on big-endian

Forwarded: https://gitlab.com/inkscape/inbox/-/issues/11895
---
 testfiles/cli_tests/CMakeLists.txt | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/testfiles/cli_tests/CMakeLists.txt b/testfiles/cli_tests/CMakeLists.txt
index d4e3d99..99e8c80 100644
--- a/testfiles/cli_tests/CMakeLists.txt
+++ b/testfiles/cli_tests/CMakeLists.txt
@@ -928,6 +928,9 @@ function(add_pdfinput_test name pages font_strategy)
 
 endfunction(add_pdfinput_test)
 
+include (TestBigEndian)
+TEST_BIG_ENDIAN(IS_BIG_ENDIAN)
+
 # CMS testing disabled because CI builder doesn't support LCMS yet.
 # add_pdfinput_test(cms 1)
 add_pdfinput_test(font-spacing 1 draw-all)
@@ -937,7 +940,9 @@ add_pdfinput_test(latex 1 draw-all)
 add_pdfinput_test(transparency-groups 1 draw-all)
 add_pdfinput_test(rotated-gradients 1 draw-all)
 add_pdfinput_test(labels-utf16 1 draw-all)
+if(NOT IS_BIG_ENDIAN)
 add_pdfinput_test(multi-page-sample all draw-all)
+endif()
 add_pdfinput_test(multiple-style-same-text-block 1 keep)
 add_pdfinput_test(gs_linewidth all draw-all)
 add_pdfinput_test(XObj_layer all draw-all)
